examples/src/examples/misc/mini-stats.mjs

```diff
@@ -211,8 +228,11 @@ async function example({ canvas, pcx }) {
                 // ensure texture is uploaded (actual VRAM is allocated)
                 texture.lock();
                 texture.unlock();
-                // @ts-ignore engine-tsd
-                app.graphicsDevice.setTexture(texture, 0);
+
+                if (!app.graphicsDevice.isWebGPU) {
+                    // @ts-ignore engine-tsd
+                    app.graphicsDevice.setTexture(texture, 0);
+                }
```

Comment:
This call doesn't work on WebGPU but I don't think it's needed to cause the allocations?